Updated default value of maxTotalRows to reflect the value in the code (#14298)
apache/druid: docs/development/extensions-core/kafka-supervisor-reference.md

| `maxRowsPerSegment`               | Integer        | The number of rows to aggregate into a segment; this number is post-aggregation rows. Handoff will happen either if `maxRowsPerSegment` or `maxTotalRows` is hit or every `intermediateHandoffPeriod`, whichever happens earlier.                                                                                                                                                                                                                                                                                                                                                                                                                   | no (default == 5000000)                                                                                      |

> | `maxTotalRows`                    | Long           | The number of rows to aggregate across all segments; this number is post-aggregation rows. Handoff will happen either if `maxRowsPerSegment` or `maxTotalRows` is hit or every `intermediateHandoffPeriod`, whichever happens earlier.                                                                                                                                                                                                                                                                                                                                                                                                              | no (default == 20000000)                                                                                    |

Reviewer: ```suggestion
| `maxTotalRows`                    | Long           | The number of post-aggregation rows to aggregate across all segments. Handoff happens when either `maxRowsPerSegment` or `maxTotalRows` is hit or every `intermediateHandoffPeriod`, whichever happens first.                                                                                                                                                                                                                                                                                                                                                                                                              | no (default == 20000000)                                                                                    |
```
What does "this number is post-aggregation rows" mean in this context? Can we say "The number of post-aggregation rows to aggregate across all segments" instead?

Author: I think it means post any rollups that user may have specified in the ingestion spec

Author: @ektravel It may be confusing to say "..post-aggregation rows to aggregate...". I was changing only the default value. The same phrase "this number is post-aggregation rows" is also being used for maxRowsPerSegment field description just before the maxTotalRows.
